hack/update/make-module-resources.sh

```diff
@@ -11,11 +11,25 @@ readonly EXISTING_RESOURCES_DELETE_PATH="../../module-resources/delete"
 readonly EXISTING_RESOURCES_APPLY_PATH="../../module-resources/apply"
 readonly HELM_OUTPUT_PATH="rendered"
 readonly NEW_RESOURCES_PATH="rendered/sap-btp-operator/templates"
+readonly RBAC_FILE_PATH="../../controllers/btpoperator_controller.go"
 
 tag=$1
 helm template $tag $CHART_PATH --output-dir $HELM_OUTPUT_PATH --values $CHART_OVERRIDES_PATH --namespace "kyma-system"
 
 trap 'rm -rf -- "temp"' EXIT
+
+declare -A rbac
+
+resource() {
+  # for now, lowercase + plural is ok but it might be neccessary to introduce a different
+  # mechanism becausethis rule doesn't apply for all GVK -> resource transformations
+  echo -n "$(yq '.kind' "$1" | tr '[:upper:]' '[:lower:]')s"
```

Comment:
important to highlight this, there are some `kinds` that don't map to `resources` this way, such as `Ingress`. But given there is no `Ingress` in btp-operator chart, it is not important to solve.

Reply:
never mind, I decided to provide a map for core resources, CRDs can be added there on demand